src/components/Axes.tsx

```diff
@@ -58,6 +62,7 @@ export const drawAxes = ({
 
   const context = canvas.getContext('2d')
   const xAxisY = height - margins.bottom
+  const xDomainWidth = xDomain[1] - xDomain[0]
```

Comment:
nice! just checking there's no chance that xDomain[1] < xDomain[0]?

Reply:
hmm good point, I think that might be able to happen. I'll update this